.github/workflows/elixir.yml

```diff
@@ -17,8 +17,8 @@ jobs:
     - name: Set up Elixir
       uses: actions/setup-elixir@v1
```

Comment:
This action is unmaintained. I should use another action. See https://github.com/actions/setup-elixir .